src/i18n/sv/common.json

```diff
@@ -355,7 +352,7 @@
     },
     "registerStatement": {
       "label": "Registerbeskrivning",
-      "url": "#"
+      "url": "https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Kymp/Tonttien%20hankinnan%20ja%20luovutuksen%20asiakasrekisteri.pdf"
```

Comment:
I think the swedish version link was posted to ticket a few minutes ago, so we can use that

Reply:
added